### DISCUSSION TITLE
Request: E3V2-BLTouch-5x5-HS-v4.2.2-v1.3.4 with reversed filament sensor

### DISCUSSION THREAD

**Discussion by kaworuseventeen** [Jyers/Marlin#1127] [Q&A] [answered]
Hello!
Im sorry if this has been asked before - I'm new to this experience.

I tried compiling this firmware myself and thought I had it squared away but when I installed it I got an error "TMC Connection error" displayed on my Ender3V2. Since I was having luck with E3V2-BLTouch-5x5-HS-v4.2.2-v1.3.4 I reverted back to it.

Really what I was trying to do is change the filament sensor to the inverse of its current setting, that's all, but I am afraid my unfamiliarity with the software has caused this TMC connector error that I do not know how to correct.
I love the 5x5 BLTouch config with the highspeed probing, but since the filament sensor is triggered when filament is in the sensor, I can't use it.

Is there a way I can ask someone to compile this same firmware but with the LOW switched to HIGH?

Anyway, thank you so much for your firmware, it has really been a lifesaver.

**Comment by Cal-LF** [comment, marked as answer]
Are you using the E3V2 config template. I use the 3x3 manual mesh and change the lines as needed. Compile and make the .bin.
You would use the BLTouch-3x3-HighSpeed configs and change the probe grid to 5 and the filament sensor to high.

**Reply by kaworuseventeen** [reply]
Hi - Thank you for the suggestion, I will re-try to compile and report back. I will verify the configs I used. I had grabbed the jyers config files and used those, just changing the ubl to linear and changing low to high, and 3 probe points to 5.

**Reply by Cal-LF** [reply]
Jyres config may have settings specific to his setup causing issues in your config.

**Reply by kaworuseventeen** [reply]
This worked for me! I am testing the sensor right now. Thank you for the suggestions.
